app/components/PageHeader/PageHeader.js

```diff
@@ -1,18 +1,14 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import InstructionsBox from '../InstructionsBox/InstructionsBox';
 
-export default function PageHeader({ instructions, }) {
-  const instructionsProps = { instructions, };
+export default function PageHeader({ children, }) {
```

Comment:
A note on default exports: many people feel strongly about avoiding using them. I didn't for a long time, but have mostly changed my mind and no longer use them. You can google and read why people dislike them.

Reply:
read about it and it does make sense not to use them, both because of the autocomplete option in IDE's and to sustain a single naming convention for of the imported objects (unless `import { foo, } as bar...` is used, but that's a whole different topic).

now, how would you suggest introducing this refactor into the project? 
- On one of the current branches? 
- On the base/master branch? 
- Creating a separate branch for this specific piece of refactoring?